torch/fx/experimental/symbolic_shapes.py

```diff
@@ -3105,24 +3117,36 @@ def _create_symbolic_sizes_strides_storage_offset(
                 else:
                     r = DimDynamic.DUCK
                 dynamic_dims.append(r)
+                dynamic_strides.append(r)
             dynamic_dims = [DimDynamic.DUCK] * dim
+            dynamic_strides = [DimDynamic.INFER_STRIDE] * dim
```

Comment:
Hi,

I noticed some confusing code logic between lines 3113 and 3120. It seems that the assignments made in these lines are overridden by the dynamic status being reset to DUCK on lines 3121/3122. Is there a specific reason for keeping the code this way?

Thank you.